### PR TITLE
[kesch] Adding ecCodes  with python 2 support

### PR DESCRIPTION
**Pull request by victorusu**

### CODE REVIEW

File: easybuild/easyconfigs/e/ecCodes/ecCodes-2.7.0-gmvolf-17.02-Python-2.7.13.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+versionsuffix='-Python-2.7.13'
+
+homepage = 'https://software.ecmwf.int/wiki/display/GRIB/Home'
+description = """ecCodes is a package developed by ECMWF which provides an
+application programming interface and a set of tools for decoding and encoding
+messages in different formats"""
+
+toolchain = {'name': 'gmvolf', 'version': '17.02'}
+
+source_urls = ['https://software.ecmwf.int/wiki/download/attachments/45757960/']
```
Comment (lucamar) [inline easybuild/easyconfigs/e/ecCodes/ecCodes-2.7.0-gmvolf-17.02-Python-2.7.13.eb]:
```
Page Not Found
```

**Comment by lucamar** [comment]
retest this please

File: easybuild/easyconfigs/e/ecCodes/ecCodes-2.7.0-gmvolf-17.02-Python-2.7.13.eb (hunk 1 above)
Comment (victorusu) [reply easybuild/easyconfigs/e/ecCodes/ecCodes-2.7.0-gmvolf-17.02-Python-2.7.13.eb]: yes, I couldn't find a solution for this problem yet. My user is able to download the code, but not jenkins

**Comment by gppezzi** [comment]
retest this please

**Comment by gppezzi** [comment]
Problem is on the routing of keschln01, which should be used exclusively by MCH. 
```
Connecting to software.ecmwf.int (software.ecmwf.int)|136.156.180.50|:443... failed: No route to host.
```

I've updated the jenkins slaves in order to avoid using ln01 and I've reported the issue to Marco Induni.

**Pull request merged**